editor/views/record_sets.py

```diff
@@ -257,6 +259,10 @@ def _render_left_panel():
             st.checkbox(
                 "The RecordSet is an enumeration",
                 key=key,
+                help=(
+                    "Enumerations indicate that the RecordSet takes its values in a"
+                    " finite set. Similar to `ClassLabel` in TFDS or Hugging Face."
```

Comment:
Maybe add a link to [the TFDS page](https://www.tensorflow.org/datasets/api_docs/python/tfds/features/ClassLabel) ?

Reply:
Done.